### ISSUE TITLE
如何返回字符识别的置信度呢

### ISSUE THREAD

**Issue by wuxiaolianggit** [chineseocr/chineseocr#265]
请问大神如何返回字符识别的置信度呢

**Comment by wenlihaoyu** [comment]
ctc实际不支持返回置信度，但是你可以在模型输出加一个F.softmax,然后解码环节把对应的概率输出即可。

**Comment by wuxiaolianggit** [comment]
好的，十分感谢 @wenlihaoyu

**Comment by clnFind** [comment]
想问下，加载模型后 输出概率，具体是怎么写的，可以有代码示例吗？

**Comment by amandazw** [comment]
> 
> 
> ctc实际不支持返回置信度，但是你可以在模型输出加一个F.softmax,然后解码环节把对应的概率输出即可。

您好，请教您一个问题。我现在双向lstm之后的tensor结果都是0.，要怎样加一个F.softmax,然后解码呢。

**Comment by amandazw** [comment]
> 
> 
> ctc实际不支持返回置信度，但是你可以在模型输出加一个F.softmax,然后解码环节把对应的概率输出即可。

您好，以下是我的输出tensor，我通过softmax之后矩阵变成了全1或者全0矩阵是怎么回事呢？我想要归一化道统一的区间好查看置信度该怎么办呢
tensor([[[-160.8315, -172.9377, -180.9977,  ..., -180.9755, -184.6772,
          -179.4818]],

        [[ -77.8825, -111.0179, -132.1096,  ..., -151.8499, -164.6233,
          -158.0727]],

        [[ -72.4325, -104.2949, -121.6171,  ..., -127.2875, -133.0346,
          -127.0531]],
        [[ -41.4288,  -93.3680, -108.5984,  ...,  -92.0508, -100.0767,
          -106.3183]],

        [[-120.5992, -167.7742, -181.6977,  ..., -170.9288, -172.7659,
          -176.7408]],

        [[-208.6236, -224.4045, -225.6789,  ..., -226.2828, -220.8472,
          -224.1825]]]
